photutils/detection/findstars.py

```diff
@@ -906,6 +907,8 @@ def find_stars(self, data):
               ``-2.5 * log10(flux)``.  The derivation matches that of
               `DAOFIND`_ if ``sky`` is 0.0.
 
+            If no stars are found then an empty table is returned.
```

Comment:
I think the critical thing here is that we can have many kinds of "empty" tables: A table that has no rows but still has valid column headers or a table that has nothing - not even column names. IMO it would be better to return the first kind of "empty" tables.

Reply:
This line does not specify the kind of emptiness that code is going to return

Reply:
@mcara "kind of emptiness" sounds very philosophical! :smile:

Reply:
The kind that `findstars` currently returns is "spooky"

Reply:
What I am trying to say is that there is a phase transition between an empty _table_ and an empty _catalog_: a catalog has a structure (i.e., provided by column names).